From 70e53605cfe83ae798ccf4027e27d36b0077ad67 Mon Sep 17 00:00:00 2001
From: Alex Moundalexis <amoundalexis@apple.com>
Date: Fri, 14 May 2021 10:50:46 -0400
Subject: [PATCH] updated copyright year on web site

---
 documentation/sphinx/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/documentation/sphinx/conf.py b/documentation/sphinx/conf.py
index 5ec9238930..ab42fdba6a 100644
--- a/documentation/sphinx/conf.py
+++ b/documentation/sphinx/conf.py
@@ -49,7 +49,7 @@
 
 # General information about the project.
 project = u'FoundationDB'
-copyright = u'2013-2018 Apple, Inc and the FoundationDB project authors'
+copyright = u'2013-2021 Apple, Inc and the FoundationDB project authors'
 
 # Load the version information from 'versions.target'
 import xml.etree.ElementTree as ET
